src/interface/src/app/treatments/apply-treatment/apply-treatment.component.scss

```diff
@@ -0,0 +1,125 @@
+@import "colors";
+
+.prescription {
+  display: block;
+  margin-bottom: 8px;
+}
+
+.form-fixed {
+  padding: 16px 16px 0;
+}
+
+.prescriptions {
+  padding: 16px;
+  overflow-y: auto;
+}
+
+.prescription-sequence {
+  display: block;
+}
+
+.label {
+  margin-bottom: 8px;
+  color: $color-dark-gray;
+}
+
+.project-area-name {
+  margin-bottom: 16px;
+}
+
+.modal-body {
+  padding: 0;
+  display: flex;
+  flex-direction: column;
+}
+
+.sequence-type {
+  width: 100%;
+
+  // all these overrides are to make the dropdown look centered and aligned
+  ::ng-deep {
+    .mat-form-field-infix {
+      border-top: 0;
+      height: 50px;
+      display: flex;
+      align-items: center;
+      padding: 0;
+    }
+
+    .mat-select-arrow-wrapper {
+      align-items: end;
+    }
+
+    .mat-form-field-outline {
+      top: 0;
+    }
+
+  }
+
+
+}
+
+sg-modal {
```

Comment:
actually a good point, maybe the header and  footer stuff should go in the modal component itself? This is just so it doesn't shrink if there is not enough space.

This component behaves a bit different than the modal, which part of the content scrollable, and part fixed. Not sure its worth moving this behavior to the modal component honestly.

Reply:
I'm also open to having a separate special component that's just for these dialogs that are within the map view.

Reply:
Do we have more than one of this though? 
I think might be useful to have more generic modals for delete/info dialogs, so we don't need to create bespoke components for each.

Reply:
Off the top of my head, I was thinking about the [Treatment Legend](https://www.figma.com/design/njHEIn6MWPYJ5hw4we0lFY/Impacts?node-id=1748-50038)
I thought there were more, but it might just be these two -- or three, if you count the Edit Treatment dialog.